source/codegen/common_helpers.py

```diff
@@ -139,6 +139,16 @@ def get_size_mechanism(parameter):
   size = parameter.get('size', {})
   return size.get('mechanism', None)
 
+def get_size_expression(parameter):
```

Comment:
note: this code was moved from `service_helpers.mako` since I needed to call it from multiple places.